04/style.css

```diff
@@ -0,0 +1,158 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+    background-color: lightblue;
+    width: 100%;
+}
+
+
+nav {
+    width: 100%;
+}
+
+.hamburger {
+    width: 40px;
+    height: 35px;
+    border: 0;
+    border-top: 5px solid blue;
+    background: transparent;
+    position: relative;
+    top: 10px;
+    right: -10px;
+    transition: 0.3s transform linear;
+}
+
+.hamburger::before,
+.hamburger::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 100%;
+    border-top: 5px solid blue;
+    transform: translateY(10px);
+}
+
+.hamburger::after {
+    border-top: 5px solid blue;
+    transform: translateY(25px);
+    transition: 0.3s transform linear;
+}
+
+.hamburger:focus {
+    transform: rotate(45deg) translateY(5px);
+    border: none;
+}
+
+.hamburger:focus::after {
+    transform: rotate(-90deg) translateX(-10px);
+}
+
+
+
+.nav__list {
+    margin-top: 50px;
+    list-style: none;
+    transform: translateX(-500px);
+    transition: transform .2s;
+    text-align: center;
+}
+
+.hamburger:focus~.nav__list {
+    transform: translateX(0);
+}
+
+
+
+.nav__item a {
```

Comment:
jeśli chcesz stylować zgodnie z BEM to każdy element powinien mieć klasę i po klasie określamy jego wygląd